core/org.wso2.carbon.server/src/main/java/org/wso2/carbon/server/extensions/DropinsBundleDeployer.java

```diff
@@ -148,6 +148,10 @@ public void perform() {
                         bundleSymbolicName = bundleSymbolicName.split(";")[0];
                     }
                 }
+                //According to the OSGi spec, Bundle-Version is an optional header; the default value is 0.0.0
+                if (bundleVersion == null) {
+                    bundleVersion = "0.0.0";
```

Comment:
IMO we should make the bundle version a constant since this value is a conceptual value. WDYT?